src/test/data/pkgDeployTestData/PkgFolder/Content/en-us/EndHtml/HTML/Default.htm

```diff
@@ -0,0 +1,56 @@
+﻿<html xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
please remove all the Content/ parts of the package folder in TestData. It doesn't (or at least, shouldn't) participate in the actual deploy part, as those files are more commonly used for AppSource publishing and deployment (which is way beyond of what this new deploy action does)